metrics/BUILD

```diff
@@ -13,17 +13,38 @@ filegroup(
     tags = ["automanaged"],
 )
 
+config_setting(  
+    name = "k8",
+    values = {"host_cpu": "k8"},
+    visibility = ["//visibility:public"],
+)
+
+config_setting(  
+    name = "darwin",
+    values = {"host_cpu": "darwin"},
+    visibility = ["//visibility:public"],
+)
+
 py_test(
     name = "bigquery_test",
     srcs = [
         "bigquery.py",
         "bigquery_test.py",
     ],
-    args = ["--jq=$(location @jq//file)"],
-    data = [
-        ":all-srcs",
-        "@jq//file",
-    ],
+    args = select({
+        ":k8": ["--jq=$(location @jq_linux//file)"],
+        ":darwin": ["--jq=$(location @jq_osx//file)"],
+    }),
+    data = select({
+        ":k8": [
+            ":all-srcs",
```

Comment:
factor `":all-srcs"` out of the `select`.

also, this should really be `glob(["configs/*"])` rather than `":all-srcs"`.

Reply:
done, had to also include `README.md`. I'm not sure if the way I wrote it is idiomatic or not.